### DISCUSSION TITLE
How to enable Vuser detailed logging

### DISCUSSION THREAD

**Discussion by qasimali80** [TPC-Council/HammerDB#756] [Q&A] [answered]
I want to see what warehouse each VUser is accessing for each transaction. I keep seeing below log. How to unsuppress the the  logging? What option to use in GUI and./or CLI to do this 

"Processing 1000000 transactions with output suppressed... ":

Thanks

**Comment by sm-shaw** [comment, marked as answer]
With the default timed workload - add a line to print the warehouse id as shown:
![warehouseselection](https://github.com/user-attachments/assets/8ff1e727-ff23-4191-83ce-e1915def3c1d)
This will then print the warehouse id selected:
![warehouseselection2](https://github.com/user-attachments/assets/85bb85ac-49de-424a-a298-d1e0dba92c90)
Or the use all warehouse option will divide all warehouses between the Vusers:
![warehouseselection3](https://github.com/user-attachments/assets/dbe8ba5c-6395-400f-a387-52a45de3fa3a)
